Project.toml

```diff
@@ -5,30 +5,41 @@ version = "0.1.0"
 [deps]
 ArgCheck = "dce04be8-c92d-5529-be00-80e4d2c0e197"
 ArraysOfArrays = "65a8f2f4-9b39-5baf-92e2-a9cc46fdf018"
+BAT = "c0cd4b16-88b7-57fa-983b-ab80aecada7e"
```

Comment:
If there's a package extension for BAT, then BAT should be in `[weakdeps]`.

Reply:
I think we don't need the extension right now. So I would just delete the extension and just leave BAT as `[deps]` as long as we can not use the DistributionTransform separately without `BAT`.  And if we wanna go to Bayesian fits at some point we anyhow need it.

Reply:
Ok